.pre-commit-config.yaml

```diff
@@ -9,30 +9,23 @@ repos:
   - id: debug-statements
   - id: mixed-line-ending
     args: [--fix=lf]
-- repo: https://github.com/pycqa/isort
-  rev: 5.12.0
+- repo: https://github.com/charliermarsh/ruff-pre-commit
+  rev: v0.0.253
   hooks:
-  - id: isort
-    args: [--profile=black]
-- repo: https://github.com/asottile/pyupgrade
-  rev: v3.3.1
-  hooks:
-  - id: pyupgrade
-    args: [--py37-plus]
+  - id: ruff
+    args: [--format, github]  # --fix, --exit-non-zero-on-fix]
```

Comment:
Why not enabling autofixer? Previously we had isort enabled, and now this behavior would be gone.

Reply:
Ruff has 500 rules and many have fixers but maintainers may not agree to all fixes so it is better to take things one step at a time but...  Let's jump into the deep end.

Reply:
--fixed ;-)